pom.xml

```diff
@@ -17,13 +17,21 @@
     <maven.compiler.target>17</maven.compiler.target>
   </properties>
 
+  
+
   <dependencies>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-api</artifactId>
       <version>5.8.2</version>
```

Comment:
please put this version in a property, e.g. <javalin.version>....

and reference it here, e.g. <version>${javalin.version}</version>